flashinfer/gemm/gemm_base.py

```
@@ -2954,8 +2954,11 @@ def _check_mm_fp4_problem_size(
     out: Optional[torch.Tensor] = None,  # unused
     block_size: int = 16,
     use_8x4_sf_layout: bool = False,  # unused
-    backend: Literal["cudnn", "trtllm", "cutlass", "auto"] = "auto",  # unused
+    backend: Literal[
```

Ruff (0.15.1)

[warning] Unused function argument: `backend`

(ARG001)